pkgs/tools/audio/tidal-dl/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, buildPythonApplication
+, fetchPypi
+, mutagen
+, requests
+, colorama
+, prettytable
+, pycrypto
+, pydub
+}:
+let
+  aigpy = buildPythonPackage rec {
```

Comment:
I thought about packaging this dependency separately, but it's just a personal library (from the same author), and the only relevant software using it seems to be this one.

Reply:
Please add the meta, specially about the license to make sure that the licenses are correct.

Reply:
I thought about it, and I think it's better for aigpy to be passed as an argument, so that it is overridable. It looked kinda bad in-lining it there, so I simply made aigpy a separate package anyway.